### ISSUE TITLE
Flags.IS_GROUNDED is always set

### ISSUE THREAD

**Issue by mwnDK1402** [GreenByteSoftware/UNet-Controller#6]
I tried the demo of this controller, and the player can jump in the air forever. I tried looking into changing this behaviour, but found this in the code:
```
hitNormal = new Vector3(0, 0, 0);

if (Vector3.Angle(Vector3.up, hitNormal) <= data.slopeLimit)
    inpRes.flags |= Flags.IS_GROUNDED;
else
    inpRes.flags &= ~Flags.IS_GROUNDED;
```

**Comment by PcJax** [comment]
That was broken on 2018.2, I have a fork with a fix for that:

https://github.com/PcJax/UNet-Controller/commit/165083ec83e235cc544936582ec69792a5962db0

**Comment by h33p** [comment]
Fixed in 64ea3df4304f16897ee44d770e9ccfb99ccb627b
